<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_uicfg</class>
 <widget class="QWidget" name="view_uicfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>view_uicfg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>新增(&amp;A)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/20070402165925169.gif</normaloff>:/ssp_gui/Resources/20070402165925169.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>删除(&amp;D)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/del.gif</normaloff>:/ssp_gui/Resources/del.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_copy">
          <property name="text">
           <string>复制(&amp;C)</string>
          </property>
          <property name="icon">
           <iconset resource="ssp_gui.qrc">
            <normaloff>:/ssp_gui/Resources/20070402165933688.gif</normaloff>:/ssp_gui/Resources/20070402165933688.gif</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_cls">
          <property name="text">
           <string>分类</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_cls"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_useCls">
          <property name="text">
           <string>分类过滤数据集选择</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>116</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="mTableWidget" name="tableWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>mTableWidget</class>
   <extends>QTableWidget</extends>
   <header>mtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ssp_gui.qrc"/>
 </resources>
 <connections/>
</ui>
